bin/pa11y.js

```diff
@@ -124,9 +132,14 @@ async function runProgram() {
 	}
 }
 
+/**
+ * Process options based on precedence rules.
+ * CLI options take precedence over config options (which take precedence over defaults)
+ * @returns {Object} An object containing all the options that have been specified
```

Comment:
You could do a `@typedef` and define the full config object....if you _really_ cared... 🤔

Reply:
I don't! :trollface: